docs2/site/docs/migrations/migration8.md

```diff
@@ -686,6 +686,21 @@ schema
 Note that you may call `LinkSchema` multiple times with the same URL to apply additional configuration
 options to the same url, or with a separate URL to link multiple schemas.
 
+### 19. `FromSchemaUrl` added to `AppliedDirective`
+
+This supports using a directive that was separately imported via `@link`. After importing the schema as described
```

Comment:
The beginning of the sentence seems to miss some words. Did you mean to say "This property supports"?

Reply:
```suggestion
This property supports using a directive that was separately imported via `@link`. After importing the schema as described
```